packaging/hudi-utilities-slim-bundle/README.md

```diff
@@ -17,6 +17,89 @@
 
 # Usage of hudi-utilities-slim-bundle
 
-Starting from versions 0.11, Hudi provides hudi-utilities-slim-bundle which excludes hudi-spark-datasource modules.
-This new bundle is intended to be used with Hudi Spark bundle together, if using hudi-utilities-bundle solely
-introduces problems for a specific Spark version.
\ No newline at end of file
+Starting from versions 0.11, Hudi provides hudi-utilities-slim-bundle which excludes hudi-spark-datasource modules. This new bundle is intended to be used with Hudi Spark bundle together, if using
+hudi-utilities-bundle solely introduces problems for a specific Spark version.
+
+## Example with Spark 2.4.7
```

Comment:
should this be moved to asf-site as part of 0.12 update?

Reply:
i thought we wanted to keep all bundle docs under packaging/<bundle>/readme.
maybe we can keep it in this readme and point to this in the release note.